views/index.py

```diff
@@ -1,7 +1,65 @@
-from flask import render_template
+from flask import render_template, request, session
 from . import library
+from models.user import User
+from forms.login_form import LoginForm
+from forms.registration_forms import RegistrationForm
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db
+from flask_login import LoginManager
```

Comment:
Let's order, group properly imports, first python standard library, then python libraries, then flask code, then app code. Fix that also in the rest of your submitted changes.